app/containers/SignIn/index.js

```diff
@@ -183,7 +198,21 @@ class SignIn extends Component {
       </Animated.View>
     );
   }
-
+  changeValues = (value) => {
+    splitted_vals = value.split(":");
+    store.dispatch(setHostUrl(splitted_vals[0]));
+    store.dispatch(setHostPort(splitted_vals[1]));
```

Comment:
Where is the store object that you are trying to access here?
Where have you imported these actions?